src/ui/interactivity/gestures/drag-outside.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Drag outside an app
+description: How to drag from an app to another app or the operating system.
+---
+
+You will likely want to implement
+drag and drop somewhere in your app.
+
+You have a couple potential approaches
+that you can take. One directly uses
+Flutter widgets and the other uses a package
+available on pub.dev.
+
+## Draggable and DragTarget
+
+If you want to implement drag and drop within
```

Comment:
We have a widget of the week video for [Draggable](https://youtu.be/q4x2G_9-Mu0) that covers the same content!